pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   logging: ^1.0.1
   path: ^1.8.0
   retry: ^3.1.0
+  eterl: ^0.0.2
```

Comment:
If eterl is not yet stable and its API might have breaking changes before 1.0.0, we should pin it's version instead of just requiring a minimum.

Reply:
The only breaking change that will come is with BigInts, as they're not well implemented. And discord does not send bigints.
But yeah, it's still good to pin.